### ISSUE TITLE
CarPlay: change podcast artwork to mirror chapter artwork

### ISSUE THREAD

**Issue by pocketcasts** [Automattic/pocket-casts-ios#3]
### Steps to reproduce the behavior

1. Play a podcast with a custom chapter art different from the podcast art
2. It doesn't display on the CarPlay display

### Additional information

Example:

<img src="https://user-images.githubusercontent.com/42823211/178276536-df542400-d60e-4229-be4d-844d1b5b3807.jpeg" width="300">

The artwork on CarPlay:

![132536129-8ed31aa4-39ab-498e-ad27-c202f8bc4eab](https://user-images.githubusercontent.com/42823211/178276730-8b03bd9b-5d2c-4e9a-a11c-0f14d687c9a4.jpeg)

Ps.: this does work as expected in Apple's Podcasts app and it doesn't seem to be car-unit-specific.

**Comment by drm87** [comment]
Would be great, if this feature would be implemented. It would make sense to bind its activation to the "publish chapter titles" options in the app's settings.

**Comment by lawruhl** [comment]
Yes, please. It seems like every other major podcast app shows chapter art in Carplay and on the iOS Lockscreen/Dynamic Island. This is a gap for PocketCasts and hope it can be made a priority!

**Comment by ArnaudK69** [comment]
Yes Update Artwork for each chapters! please!

**Comment by cecebe** [comment]
I need this as well. So many German podcast have chapter artworks and it would be good to show it in the carplay nps

**Comment by kean** [comment]
Fixed in https://github.com/Automattic/pocket-casts-ios/pull/4596
